client/svelte.config.js

```diff
@@ -11,6 +11,7 @@ const config = {
     adapter: adapter(),
     alias: {
       $components: "./src/components",
+      $stores: "./src/store",
```

Comment:
it seams like some people the internet use `$lib/stores` instead of `$stores` but either way seams fine to me

Reply:
we decided we like `$stores`